test/_init_tests.js

```diff
@@ -76,7 +77,12 @@ sinon.sandbox.create = function(config) {
 // Global cleanup of tags added during tests. Cool to add more
 // to selector.
 afterEach(() => {
-  const cleanup = document.querySelectorAll('link,meta');
+  const cleanupTagNames = ['link', 'meta'];
+  if (platform.isChrome()) {
```

Comment:
It was, Github sometimes doesn't seem to update the diff view for some reason. Now updated.